cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -1957,7 +1957,8 @@ private int getRandomExclude(int min, int max, int exclude) {
         if (max - min < 2)
             return min;
         for (int i = 0; i < 10; i++) {
-            val = rnd.nextInt() % (max - min) + min;
+            /* next(31) ensures non-negative */
+            val = rnd.next(max - min) + min;
```

Comment:
```suggestion
            val = rnd.nextInt(max - min) + min;
```

Reply:
You dont want anything to a number of bits - you want an integer up to a value

Reply:
You're right!